### ISSUE TITLE
Lasso problem on Mac OS X

### ISSUE THREAD

**Issue by ilkayisik** [ofgulban/segmentator#35]
Lasso controls are glitchy and cannot be switched off on Mac.
Might be related to matplotlib backend.

**Comment by ofgulban** [comment]
@MSchnei, would you be willing to also check this problem? Whether it occurs for you or not?

**Comment by MSchnei** [comment]
I just checked and had the same issue. With every arrow press it writes to command line but the wedge does not move. This potentially results from segmentator not considering the figure an active window but I am not sure

**Comment by ofgulban** [comment]
@MSchnei @ingo-m any updates on this? Is the problem still there?

**Comment by ofgulban** [comment]
@ingo-m @MSchnei Could you check this problem at some point? Does is still exist after v1.3.0 on Mac?

**Comment by ingo-m** [comment]
Ok I will check this.

**Comment by ingo-m** [comment]
I tested this with the following system/environment configuration:

| Package    | Version |
|------------|---------|
| Mac OS     | 10.12.6.|
| numpy      |  1.11.3 |
| scipy      |  0.19.0 |
| matplotlib |   2.0.2 |
| nibabel    |   2.1.0 |

I was able to activate the Lasso, select a region in the 2D histogram, deactivate the Lasso again, and move the circle mask (is it still called like that?) afterwards. Moreover, I was able to export the label to nii. So the Lasso feature seems to be functional.

However, I noticed that when hitting the `Reset` button afterwards, the Lasso region does not disappear (the circle mask moves to the origin though, so the button is active). Is this behaviour expected?

**Comment by ofgulban** [comment]
This behavior is expected. Reset button resets depending on the mode. Switching back to lasso and pressing reset should clean the histogram. Do you think we should address this? Maybe we can change the reset button label depending on the mode (e.g. 'Reset Circle' -> 'Reset Lasso')

**Comment by ingo-m** [comment]
Yes I think this is a good idea. The present behaviour is good and does not have to be changed. If the text in the button changes depending on mode as you suggested, this will make the user aware of the logic behind it. (Also, this is better than adding another button to avoid clutter.)
